### PR TITLE
Add Scrapbox Extension

### PR DESCRIPTION
**Pull request by zakuni**
## Description

A new extension to search pages in [Scrapbox.io](https://scrapbox.io ), which is a wiki system by Nota inc., users can create their own wiki project.
Results pages can be opened via browser.

## Screencast

scrapbox-1
scrapbox-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/scrapbox/src/components/Pages.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <List>
+      {props.pages?.map((page: any) => (
+        <List.Item
+          key={page.id}
+          icon="list-icon.png"
+          title={page.title}
+          actions={
+            <ActionPanel>
+              <Action.OpenInBrowser url={encodeURI(`https://scrapbox.io/zakuni/${page.title}`)}></Action.OpenInBrowser>
```
Comment (pernielsentikaer) [inline extensions/scrapbox/src/components/Pages.tsx:13]: Is this hardcoded to your scrapbox?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

I left a single comment that we need to ensure before proceeding, could you have a look 😄 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/scrapbox/src/components/Pages.tsx (hunk 1 above)
Comment (zakuni) [reply extensions/scrapbox/src/components/Pages.tsx:13]: Oops, yes, I've mistakenly left a hardcoded link to my scrapbook. I'm going to fix it.
Thanks for your review!

Comment (zakuni) [reply extensions/scrapbox/src/components/Pages.tsx]: @pernielsentikaer Hi, actually `Pages.tsx` itself was the old unused file I'd forgotten to remove. I've already deleted the file and pushed the branch. 
Would you please review again? Thanks.

**Comment by zakuni** [comment]
@pernielsentikaer Hi, should I take any further actions for it? I know snyk tests have failed, but since I don't have a permission to see the details, I think I need someone's help to fix it.

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* Is there another way to fetch users' projects? It would be if it was possible to fetch a list of projects and use it as a dropdown instead, I can imagine that it's hard to remember all projects. But it might not be possible to do.

* Could the keyword be optional?

* If no matches it just keeps spinning, it would be nice to handle it with a toast and return an empty array 😊

image

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Pull request closed without merging**